central/testutils/testexport.go

```diff
@@ -0,0 +1,348 @@
+package testutils
```

Comment:
Let's just name this file `export` instead, `test` is already inferred by the package name.